capellambse/aird/parser/__init__.py

```diff
@@ -20,15 +20,16 @@
 from lxml import etree
 
 import capellambse
+from capellambse import _namespaces as _n
```

Comment:
In other places we use `import collections.abc as cabc`, it would be good to unify this to use the same format:

```suggestion
import capellambse._namespaces as _n
```